lib/foreman_google/engine.rb

```diff
@@ -13,6 +13,7 @@ class Engine < ::Rails::Engine
     initializer 'foreman_google.register_plugin', before: :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_google do
         requires_foreman '>= 2.4.0'
+        register_global_js_file 'global'
```

Comment:
Why do we need it globally? Isn't it enough to put it in asset pipeline and require from the erb?

Reply:
I tried to require the js file through webpack or asset pipeline to the `_gce.erb` partial by adding - 
```ruby
javascript 'foreman_google/form' # assets pipeline
-------
javascript *webpack_asset_paths('foreman_google', :extension => 'js') # webpack
```
but with no success, I guess because the `<script>` tags are escaping inside the `form_for`. 
Any idea? or should we keep the gloabl?